elixir/apps/domain/lib/domain/resources/resource/query.ex

```diff
@@ -25,14 +25,17 @@ defmodule Domain.Resources.Resource.Query do
   end
 
   def by_authorized_actor_id(queryable \\ not_deleted(), actor_id) do
-    subquery = Domain.Policies.Policy.Query.by_actor_id(actor_id)
+    subquery =
+      Domain.Policies.Policy.Query.by_actor_id(actor_id)
+      |> where([policies: policies], policies.resource_id == parent_as(:resources).id)
+      |> limit(1)
 
     queryable
     |> join(
-      :inner,
+      :inner_lateral,
       [resources: resources],
       policies in subquery(subquery),
-      on: policies.resource_id == resources.id,
+      on: true,
```

Comment:
@AndrewDryga I think this may have created a bug where I'm only seeing one resource?

<img width="267" alt="Screenshot 2023-12-08 at 7 08 23 AM" src="https://github.com/firezone/firezone/assets/167144/dff02bd7-f790-4aa7-b10d-b037cae1b477">